src/main/java/io/personium/client/DavCollection.java

```diff
@@ -269,6 +269,15 @@ public ServiceCollection service(String name) {
         return new ServiceCollection(this.accessor, UrlUtils.append(this.getPath(), name));
     }
 
+    /**
+     * This method requests the GET method for the DAV collection itself by calling its overloaded version.
+     * @return GET Response int string form
+     * @throws DaoException Exception thrown
+     */
+    public String getString() throws DaoException {
```

Comment:
I think it is better to add interfece `getMetaData` to the BoxManager class to retrieve box meta data, .